### PR TITLE
Adds knight armour

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Adds three sets of classic medieval armour, with the red/black variants sprited by CosmicScientist and the Templar variant done by me (based off his sprites). They have riot armour values (good against melee, bad against everything else).

Puts a suit of the templar armour in the chaplain's locker so he can run around yelling about crusades.

> muh game balance my roleplay job

I don't think a single suit of riot armour (5 whole points in cargo! No defense against anything but melee!) is going to kill game balance, and the chaplain is gameplay critical against revenants/HoG/cult, so we gave up on him being a throwaway civilian job ages ago.

https://tgstation13.org/phpBB/viewtopic.php?f=11&t=3735

Sprites are there for those who wish to see them.

:cl: Kor and Ausops
rscadd: Added four different suits of medieval plate armour.
rscadd: The Chaplain starts with a suit of Templar armour in his locker. God wills it!
rscadd: Rumour has it that Nanotrasen is now training a new elite unit of soldiers to deal with paranormal threats. Corporate was unable to be reached for comment.
/:cl:

deus vult

deus vultx2

**To do:**
- [ ] New item sprites
- [ ] Better from-behind sprites on the helmet
- [ ] Better shading on the red & gold tabard
- [ ] death to pillow shading
- [ ] Give the armor some shine

**Bonus Level**
- [x] Have ausops resprite the whole thing

Also added paranormal ERT hardsuits, sprited by Ausops. They're the same as normal hardsuits but they are fireproof and have an inbuilt null rod. I'll create a button to actually spawn the paranormal ERT in another PR.

### CODE REVIEW

**Comment by Anonus** [comment]
Chaplain memes recommence.

**Comment by optimumtact** [comment]
crusader image

**Comment by KorPhaeron** [comment]
Should I add a button to the templar armour that makes your character shout "GOD WILLS IT!!!" ?

**Comment by optimumtact** [comment]
Yes please!

**Comment by PKPenguin321** [comment]
:+1: reasoning for this is pretty good, especially considering HoG and how in that mode claymores can 3 shot a chaplain who is vital to the round

**Comment by daman997** [comment]
KNIGHT BATTLES FOR DAYS

**Comment by WJohn** [comment]
I am not at all a fan of the sprites. It looks extremely dull (lacking any shine that defines metals), suffers badly from pillow shading, lacking in defining contrast between individual shades (it all blurs together like a gradients, that's bad). Outlines are much too close to the brightness of the fill.

Crusador armor is more tolerable, at least. Looks like cloth (maybe leather). Appears to be a missing dixel around the crotch there (might be intended). It's a bit hard to tell if this is actual armor or just robes. If it's supposed to be armor with sheets draping over them, then it's not obvious enough that that's going on.

**Comment by KorPhaeron** [comment]
WJ is trying to stop the crusade

**Comment by MrPerson** [comment]
The sprites really are pretty bad

**Comment by Dorsisdwarf** [comment]
DEUS VULT

**Comment by Aranclanos** [comment]
include the sprites in the OP
and fix them until WJ accepts them

**Comment by Stenthus** [comment]
sprite -
letting chaplains be crusaders +++++++

the awkward thing is that the chap's melee weapon of choice is (most of the time) going to be their null rod which doesn't really theme well with a suit of armor

**Comment by Iamgoofball** [comment]
Resprite option to make it a quarterstaff?

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -167,6 +167,22 @@
...
 
+/obj/item/clothing/head/helmet/knight
+	name = "medieval helmet"
+	desc = "A classic metal helmet."
+	icon_state = "knight_red"
+	item_state = "knight_red"
+	armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	strip_delay = 80
```
Comment (Dorsisdwarf) [inline code/modules/clothing/head/helmet.dm]: Why is the knight armor a child of riot armor, but the helmet not a child of the riot helmet?

Comment (KorPhaeron) [inline code/modules/clothing/head/helmet.dm]: Helmet toggle

Comment (Dorsisdwarf) [inline code/modules/clothing/head/helmet.dm]: Cannot flip up visor/10

**Comment by KorPhaeron** [comment]
Anyway PR is dead unless cosmic resprites stuff

**Comment by Dorsisdwarf** [comment]
I see what QJ means
The torso is just a flat slab of color

**Comment by Dorsisdwarf** [comment]
Reopened because I just spent 15 minutes copying up the URLs of the image links and putting them in your post fuck closing it so soon

**Comment by KorPhaeron** [comment]
>I just spent 15 minutes copying up the URLs of the image links 

Seems like a massive waste of time when there was already a link to all of them in the OP

**Comment by Dorsisdwarf** [comment]
Yeah but people hate clicking links.

**Comment by Ausops** [comment]
im fucking sure i made a set of armor sprites once but they got lost somewhere give me a day

**Comment by KorPhaeron** [comment]
@Ausops 

Gimme the grey knight ones as well I'll do it all in one PR

**Comment by KorPhaeron** [comment]
Shit wrong button

**Comment by Dorsisdwarf** [comment]
deus

Here have a quick paint helmet

**Comment by KorPhaeron** [comment]
Keep in mind the current crusader armour isn't in the OP

**Comment by Jalleo** [comment]
Overlays to reduce the file size and such?

**Comment by optimumtact** [comment]
DEUS VULT

**Comment by Alek2ander** [comment]
Ad majorem Dei gloriam

**Comment by Bawhoppen** [comment]
Battle standards and war flags when?

**Comment by KorPhaeron** [comment]
@Dorsisdwarf 

Any plants for onmob or finishing the suits?

**Comment by Ausops** [comment]
https://imgur.com/2bdbGSG

**Comment by Iamgoofball** [comment]
DEUS VOLT AUSOPS

**Comment by WJohn** [comment]
That's fantastic looking, ausops. Seriously amazing work. :+1: x :100:  from me.

**Comment by Thunder12345** [comment]
DEUS VULT!

**Comment by Dorsisdwarf** [comment]
On the one hand they're beautiful.

On the other hand I can't help but point out that they're not crusader kit at all

**Comment by KorPhaeron** [comment]
I'll desecrate his sprites with tabbard myself when I get home

**Comment by Dorsisdwarf** [comment]
Hah

**Comment by KorPhaeron** [comment]
Nice. Let me know when back/side sprites are done

**Comment by CosmicScientist** [comment]
I didn't even know this was a thing until today. Thanks for the @CosmicScientist, @memo in IRC or asking for better sprites in the forum thread.

**Comment by CosmicScientist** [comment]
:+1: to better sprites. /me crawls into a hole to die for his sins of pathetic spriting

**Comment by Bawhoppen** [comment]
the original sprites look like something from old goon

**Comment by duncathan** [comment]
DEVS VVLT

**Comment by KorPhaeron** [comment]
New sprites are in and WJ approves of them.

Please remove the DNM tag

**Comment by Dorsisdwarf** [comment]
DEUS

**Comment by Dorsisdwarf** [comment]
@duncathan abort abort get in the bus nerd

**Pull request merged**